docker-hub/oci-artifacts.md

```diff
@@ -0,0 +1,175 @@
+---
+title: OCI artifacts on Docker Hub
+keywords: oci, artifacts, docker hub
+description: >
+  You can use Docker Hub to store software artifacts packaged as OCI artifacts.
```

Comment:
I'd probably rephrase "store software artifacts" to simply "store any type of content"  🤷‍♂️ The advantage of this would be we wouldn't repeat `artifacts` twice in the same sentence. But feel free to ignore me -- I'm just nitpicking here 😇